RetailCoder.VBE/UnitTesting/TestEngine.cs

```diff
@@ -76,14 +77,18 @@ public void Run(IEnumerable<TestMethod> tests)
                         continue;
                     }
 
+                    var stopwatch = new Stopwatch();
+                    stopwatch.Start();
+
                     Run(testInitialize);
                     test.Run();
+                    Run(testCleanup);
+
+                    stopwatch.Stop();
+                    test.Result.SetDuration(stopwatch.ElapsedMilliseconds);
```

Comment:
doesn't this make the `IHostApplication.TimedMethodCall` method obsolete/irrelevant?